test/gui/shared/steps/steps.py

```diff
@@ -873,6 +910,27 @@ def step(context, resource, content):
     waitForFileToBeSynced(context, resource)
 
 
+@When('the user tries to overwrite the file "|any|" with content "|any|"')
+def step(context, resource, content):
+    waitForFileToBeSynced(context, resource)
+    waitForFolderToBeSynced(context, '/')
+
+    # overwriting the file immediately after it has been synced from the server seems to have some problem.
+    # The client does not see the change although the changes have already been made thus we are having a race condition
+    # So for now we add 5sec static wait
+    # an issue https://github.com/owncloud/client/issues/8832 has been created for it
+    snooze(5)
```

Comment:
do we need to wait for at least 5 sec?

Reply:
Just copy pasted from code from above. Have to try deleting it.